src/Bowerphp/Config/Config.php

```diff
@@ -250,4 +265,16 @@ protected function getHomeDir()
 
         return rtrim($home, '/');
     }
+
+    protected function getBowerrcPath()
```

Comment:
Can you please do the following?
* add a phpdoc block
* avoid do/while
* fix indent